tests/test_tmaUtils.py

```diff
@@ -338,6 +341,25 @@ def test_helperFunctions(self):
         self.assertEqual(slews, foundSlews)
         self.assertEqual(tracks, foundTracks)
 
+    @vcr.use_cassette()
+    def test_getEvent(self):
+        # test the singular event getter, and what happens if the event doesn't
+        # exist for the day
+        eventMaker = self.tmaEventMaker
+        events = eventMaker.getEvents(self.dayObs)
+        nEvents = len(events)
+
+        event = eventMaker.getEvent(self.dayObs, 0)
+        self.assertIsInstance(event, TMAEvent)
```

Comment:
Not sure if the API supports it, but could you add `self.assertEqual(event, events[0])` ?
And similar for `events[100]`?

Reply:
Oh, it looks like past-Merlin actually did implement `__eq__` for these objects, so yes, I'll do that - nice!